lib/msf/core/db_manager/import.rb

```diff
@@ -146,7 +145,7 @@ def import(args={}, &block)
         if serv.nil? && !second_pass_services.empty?
           second_pass_services.each do |service|
             next unless mports[service.port]
-            if (match_names - mports[service.port].keys).count < match_names.count
+            if (match_vulns - mports[service.port]).size < matched_vulns.size
```

Comment:
I think just this line just needs the same rename applied like the above line, then I think that's everything to get this landed 🎉